### PR TITLE
[script.module.dropbox] 8.4.0

### PR DESCRIPTION
**Pull request by micahg**
### Description
Updates to the dropbox v2 api (python sdk 8.4.0).

### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [script.foo.bar] v1.0.0

### CODE REVIEW

File: script.module.dropbox/resources/utils.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    """Show Notification
+
+    Args: 
+        title: has to be unicode
+        message: has to be unicode
+        time: Time that the message is beeing displayed
+    """
+    __addoniconpath__ = os.path.join(addon_dir(),"icon.png")
+    log(u'Notification. %s: %s' % (title, message) )
+    xbmc.executebuiltin(encode('Notification("' + title + '","' + message + '",'+(str(time)).decode('utf-8')+',"' + __addoniconpath__ + '")'))
```
Comment (Lunatixz) [inline script.module.dropbox/resources/utils.py]: Not mandatory; you might want to consider using `xbmcgui.Dialog().notification`
https://codedocs.xyz/xbmc/xbmc/group__python___dialog.html#ga27157f98dddb21b44cc6456137977aa2

File: script.module.dropbox/resources/utils.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    """
+    return __Addon.getAddonInfo('path').decode('utf-8')
+
+def encode(string):
+    return string.encode('UTF-8','replace')
+
+def decode(string):
+    return string.decode('UTF-8') 
+
+def log(message,loglevel=xbmc.LOGNOTICE):
```
Comment (Lunatixz) [inline script.module.dropbox/resources/utils.py]: Can you please set a minimum log level of LOGDEBUG.

File: script.module.dropbox/resources/language/German/strings.xml [hunk 3]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
```
Comment (Lunatixz) [inline script.module.dropbox/resources/language/German/strings.xml]: xml strings are outdated, please move to the new language system.
http://kodi.wiki/view/language_support

File: script.module.dropbox/resources/language/English/strings.xml [hunk 4]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
```
Comment (Lunatixz) [inline script.module.dropbox/resources/language/English/strings.xml]: xml strings are outdated, please move to the new language system.
http://kodi.wiki/view/language_support

File: script.module.dropbox/resources/utils.py (hunk 2 above)
Comment (micahg) [reply script.module.dropbox/resources/utils.py]: Hi -- I've made all the requested (mandatory) changes.

Thanks in advance.

Comment (Lunatixz) [reply script.module.dropbox/resources/utils.py]: minimum log level of LOGDEBUG.

File: script.module.dropbox/changelog.txt [hunk 5]
```diff
@@ -1,6 +1,8 @@
 [B]Version 8.4.0[/B]
```
Comment (Lunatixz) [inline script.module.dropbox/changelog.txt]: Kodi Krypton uses the `<news>` addon.xml parameter as a changelog. Changelog.txt is no longer supported.

**Comment by micahg** [comment]
OK, thanks @Lunatixz -- news added, changelog removed, and commits squashed.

**Comment by Lunatixz** [comment]
LGTM

**Pull request merged**